doc/api/fs.md

```diff
@@ -1549,6 +1549,23 @@ Synchronous version of [`fs.readFile`][]. Returns the contents of the `file`.
 If the `encoding` option is specified then this function returns a
 string. Otherwise it returns a buffer.
 
+_Note: The behavior of `fs.readFileSync()` is platform specific. As such,
```

Comment:
I think we should be more general about what functions are affected since it would also include the async counterpart as well as anyone using `fs.open('/path/to/dir')` with `fs.read()` or their sync counterparts.